onnxruntime/test/python/quantization/test_qdq_loss_debug.py

```diff
@@ -318,6 +319,12 @@ def test_create_weight_matching_per_channel(self):
             dq_array = matched_weights[weight_name]["dequantized"]
             self.assertEqual(float_array.shape, dq_array.shape)
 
+    def test_none_test(self):
```

Comment:
nit: naming: test_thing_does_thing_when_...

e.g. `test_create_activation_matching_handels_non_scalar_np_arrays` etc.

The current name also makes an assumption on the implementation (which we may want to avoid)